src/window.cpp

```diff
@@ -2761,8 +2761,10 @@ static void HandleKeyScrolling()
 	if (_dirkeys && !EditBoxInGlobalFocus()) {
 		int factor = _shift_pressed ? 50 : 10;
 
-		/* Key scrolling stops following a vehicle. */
-		GetMainWindow()->viewport->follow_vehicle = INVALID_VEHICLE;
+		if (_game_mode != GM_MENU && _game_mode != GM_BOOTSTRAP) {
```

Comment:
The arrow keys already do nothing when we are in the menu. (Only moving the mouse stops camera following).

Reply:
Yes, the contents of the if are not required for the GM_MENU and GM_BOOTSTRAP modes.

Reply:
Right, my point was that this check isn't required since GM_MENU is already handled.

Reply:
The title game panning does not use `follow_vehicle`. It is not required to reset it. This has symmetry with the corresponding condition in ScrollMainViewport.

An equivalent condition of `_game_mode == GM_NORMAL || _game_mode == GM_EDITOR` would be better from first principles, but this is inconsistent with the existing conditions of this type.

Reply:
Aha, thanks. I guess I should have asked why you added the check instead of arguing against it. 😅